src/workerfarm/WorkerFarm.js

```diff
@@ -268,13 +256,8 @@ class WorkerFarm extends EventEmitter {
       return parseInt(process.env.PARCEL_WORKERS, 10);
     }
 
-    let cores;
-    try {
-      cores = require('physical-cpu-count');
-    } catch (err) {
-      cores = os.cpus().length;
-    }
-    return cores || 1;
+    let cores = os.cpus().length;
```

Comment:
hmm, last time I tried this, it was much slower than using only the physical cpu count. why did you change it?

Reply:
Originally I changed it because I made the workers start on request, but that appeared to add about a second in build time. So I went back to starting all workers at the start of the workerfarm, which made it as fast as it originally was.
It should not differ that much os.cpus or real cpu count, I'd have to do some performance testing to make sure this doesn't really impact performance.

Reply:
Appears to be faster with `physical-cpu-count` as you said.

I've changed it back